CMakeLists.txt

```diff
@@ -17,32 +17,10 @@ set(CMAKE_C_EXTENSIONS ON)
 # Enable checks
 option(ENABLE_CHECKS "Enable checks using clang-tidy" OFF)
 if (${ENABLE_CHECKS})
-  find_program(CLANG_TIDY clang-tidy)
-  if (CLANG_TIDY)
-    set(CMAKE_CXX_CLANG_TIDY clang-tidy --config-file=${CMAKE_CURRENT_SOURCE_DIR}/.clang-tidy)
-  endif ()
+    find_program(CLANG_TIDY clang-tidy)
```

Comment:
Changing only the indentation... Should we keep 2-spaces (this is the current setting) indentation or 4-spaces?